packages/sol-cov/src/coverage_subprovider.ts

```diff
@@ -35,11 +43,15 @@ export class CoverageSubprovider extends Subprovider {
             this._getContractCodeAsync.bind(this),
         );
     }
-    public handleRequest(
-        payload: Web3.JSONRPCRequestPayload,
-        next: NextCallback,
-        end: (err: Error | null, result: any) => void,
-    ) {
+    /**
+     * Write the test coverage results to a file in Istanbul format.
+     */
+    public async writeCoverageAsync(): Promise<void> {
+        await this._coverageManager.writeCoverageAsync();
+    }
+    // This method must conform to the web3-provider-engine interface
+    // tslint:disable-next-line:prefer-function-over-method underscore-private-and-protected
+    private handleRequest(payload: Web3.JSONRPCRequestPayload, next: NextCallback, end: ErrorCallback) {
```

Comment:
Why did you make it private? Not that it matters as long as provider engine is in JS, but I find it confusing.

Reply:
I made it private so it wouldn't clutter the docs page... but will make it public again because I agree with you that it is part of a subprovider's public interface. I will make it clear that it is not to be called directly in the method comment.